blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -201,6 +229,27 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
         }
     }
 
+    private String prepareErrorResponse(String tenantId, String message) {
```

Comment:
Can this be placed in DefaultHandler, perhaps? Because I will need to return some 415 errors from other classes.